### ISSUE TITLE
the best epoch

### ISSUE THREAD

**Issue by Chevalier1024** [bekou/multihead_joint_entity_relation_extraction#6]
Firstly，thank you very much for your great job. I have some questions and hope to get your answer.
>1.  Train on the training set and evaluate on the dev set to obtain early stopping epoch `python3 train_es.py` 
> 2. Train on the concatenated (train + dev) set and evaluate on the test set until either (1) the max epochs or (2) the early stopping limit (specified by train_es.py) is exceeded `python3 train_eval.py` 

In the first step, we can get the best epoch based on the dev set. In my opinion, the best epoch may not be the best in the second step, because the training set change(train + dev). Another question, how do you get a stable result? I run many times and the result is different.
Looking forward for your reply, thank you!

**Comment by bekou** [comment]
Indeed, there might be a better epoch in the train + dev set, but I would like to treat the best epoch as a hyperparameter. 
Concerning your second question, I am able to get always the same results. I think that you should run the code with the exact same requirements as reported in the readme.
